README.md

```diff
@@ -23,6 +25,10 @@ Inteli OpenAPI service for interacting with the DSCP (Digital Supply-Chain Platf
 | AUTH_AUDIENCE                |    N     |                    `inteli-dev`                     | Identifier of the Auth0 API                                                          |
 | AUTH_ISSUER                  |    N     |           `https://inteli.eu.auth0.com/`            | Domain of the Auth0 API `                                                            |
 | AUTH_TOKEN_URL               |    N     |      `https://inteli.eu.auth0.com/oauth/token`      | Auth0 API endpoint that issues an Authorisation (Bearer) access token                |
+| AUTH_TEST_USERNAME           |    N     |                          -                          | Username of the auth0 user for testing                                               |
+| AUTH_TEST_PASSWORD           |    N     |                          -                          | Password of the auth0 user for testing                                               |
+| AUTH_TEST_CLIENT_ID          |    N     |                          -                          | Client ID of the auth0 application for testing                                       |
+| AUTH_TEST_CLIENT_SECRET      |    N     |                          -                          | Client secret of the auth0 application for testing                                   |
```

Comment:
Should this be here or should we create a separate section for testing envars?

Reply:
will add in new testing section